resources/views/pages/home.ejs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% include ../partials/head %>
+    <link rel="stylesheet" href="/resources/css/home.css" />
+  </head>
+  <body class="container">
+    <header>
+      <% include ../partials/header %>
+    </header>
+
+    <main class="home">
+      <div class="offer-banner">
```

Comment:
Invalid BEM, create BEM for Carousel. E.g. 
.carousel > (ul.carousel__items > li.carousel__item > img.carousel__image) + div.carousel__navigation.carousel__navigation--prev + div.carousel__navigation.carousel__navigation--next + ul.carousel_pagination > li.carousel__pagination__dot

Reply:
done